src/coreclr/jit/flowgraph.cpp

```diff
@@ -4088,6 +4088,7 @@ BasicBlock* Compiler::fgAddCodeRef(BasicBlock* srcBlk, unsigned refData, Special
 
     newBlk->bbFlags |= BBF_IMPORTED;
     newBlk->bbFlags |= BBF_DONT_REMOVE;
+    newBlk->bbFlags |= BBF_SHARED_THROW;
```

Comment:
Add the flag to `BasicBlock::dspFlags()`

Reply:
Not needed anymore.